Makefile

```diff
@@ -23,7 +23,8 @@ init:
 	GO111MODULE=on go mod vendor
 	sleep 3
 
-deps: clean
+deps:
+	rm -rf ./go.sum
```

Comment:
not `rm ./go.sum`?

Reply:
fixed in: [214b74f](https://github.com/AthenZ/authorization-proxy/pull/17/commits/214b74f445f06a47d07ce6b11f97fee140334c3d)